packages/bot-web-ui/src/components/load-modal/__tests__/recent-footer.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render, screen, waitFor } from '@testing-library/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import userEvent from '@testing-library/user-event';
+import { mock_ws } from 'Utils/mock';
+import RootStore from 'Stores/root-store';
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import RecentFooter from '../recent-footer';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(),
+}));
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({
+    saveRecentWorkspace: jest.fn(),
+    unHighlightAllBlocks: jest.fn(),
+}));
+jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
+
+window.Blockly = {
+    derivWorkspace: { asyncClear: () => ({}) },
+    Xml: { domToWorkspace: () => ({}), textToDom: () => ({}) },
+};
+
+describe('RecentFooter', () => {
+    let wrapper: ({ children }: { children: JSX.Element }) => JSX.Element, mock_DBot_store: RootStore | undefined;
+
+    beforeAll(() => {
+        const mock_store = mockStore({});
+        mock_DBot_store = mockDBotStore(mock_store, mock_ws);
+
+        wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock_store}>
+                <DBotStoreProvider ws={mock_ws} mock={mock_DBot_store}>
+                    {children}
+                </DBotStoreProvider>
+            </StoreProvider>
+        );
+    });
+
+    it('should render RecentFooter', () => {
+        const { container } = render(<RecentFooter />, { wrapper });
+        expect(container).toBeInTheDocument();
+    });
+
+    it('should render button with Open text', () => {
+        render(<RecentFooter />, { wrapper });
+        const openButton = screen.getByText('Open');
+        expect(openButton).toBeInTheDocument();
+        expect(mock_DBot_store?.load_modal?.is_open_button_loading).toBeFalsy();
+    });
+
+    it('should render import message', async () => {
```

Comment:
Can this be `should render toast message and open load modal on open button click`?

Reply:
Updated the expect of test slightly and changed the naming